lib/rubocop/cop/performance/squeeze.rb

```diff
@@ -46,7 +46,7 @@ def on_send(node)
             message = format(MSG, current: bad_method, prefer: good_method)
 
             add_offense(node.loc.selector, message: message) do |corrector|
-              string_literal = to_string_literal(replace_str)
+              string_literal = to_string_literal(replace_str.dup)
```

Comment:
Can you add a FIXME comment? e.g.:

```suggestion
              # FIXME: When requiring RuboCop 1.70.0 and above,
              # `dup` in `replace_str.dup` becomes unnecessary, so please remove it.
              string_literal = to_string_literal(replace_str.dup)
```